gpytorch/functions/_inv_matmul.py

```diff
@@ -5,6 +5,13 @@
 from .. import settings
 
 
+def _solve(lazy_tsr, rhs, preconditioner):
+    if settings.fast_computations.solves.on() or lazy_tsr.matrix_shape.numel() <= settings.max_cholesky_numel.value():
```

Comment:
`lazy_tsr.matrix_shape.numel()` will be 256 when `self` is 16x16 -- I think we should compare `lazy_tsr.size(-1)`

Reply:
Sure - I'm just using the existing `max_cholesky_numel` flag that existed for `root_decomposition`.